coccinelle/run-coccinelle.sh

```diff
@@ -67,7 +60,7 @@ for script in "${SCRIPTS[@]}"; do
     # when paired together with parallel
     #
     # 2) Make sure spatch can find our includes via -I <dir>, similarly as we do when compiling stuff.
-    #    Also, include the system include path as well, since we're not kernel and we make use of stdlib
+    #    Also, include the system include path as well, since we're not kernel and we make use of the stdlib
```

Comment:
Not a matter, but this change in the third commit should be in the second.

Reply:
Whoops, indeed.